cluster-scope/overlays/nerc-ocp-prod/feature/custom-routes/cluster-ingress.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: config.openshift.io/v1
+kind: Ingress
+metadata:
+  name: cluster
+spec:
+  domain: apps.shift.nerc.mghpcc.org
+  componentRoutes:
+    - name: console
+      namespace: openshift-console
+      hostname: console.apps.shift.nerc.mghpcc.org
+    - name: downloads
+      namespace: openshift-console
+      hostname: downloads.apps.shift.nerc.mghpcc.org
+    - name: oauth-openshift
```

Comment:
Note that every time we change the oauth url, we also need to update any oauth application to which we're linked (e.g., Github, Keycloak), because the callback URL is part of the configuration.

I would probably just leave the oauth URL alone, since it's not something that we expect end users to care about...but I don't feel strongly about it.